### ISSUE TITLE
Cocoapods 1.10.0 generates invalid scripts for static framework

### ISSUE THREAD

**Issue by till0xff** [CocoaPods/CocoaPods#10206]
# Report

## What did you do?

I have local pod with static framework (`s.static_framework = true`) which contains resource (xib or storyboard) with dot-separated name (e.g. `View.Suffix.xib`). Cocoapods 1.10.0 generates script `Pods-Bug_Example-resources.sh` with invalid resource name: `View.nib` instead of `View.Suffix.nib`.

With cocoapods 1.9.3 the error is not reproduced.

## What did you expect to happen?

Build successful

## What happened instead?

Build failed with error

```
error: Resource "/Users/till0xff/Library/Developer/Xcode/DerivedData/Bug-cmbxgygdohzlgagwgynzdhhoicou/Build/Products/Debug-iphonesimulator/Bug/Bug.framework/View.nib" not found. Run 'pod install' to update the copy resources script.
```

## Project that demonstrates the issue

 https://github.com/till0xff/cocoapods-1.10.0-bug-example.git

**Comment by dnkoutso** [comment]
@igor-makarov I believe this change https://github.com/CocoaPods/CocoaPods/commit/842dceabaa715df0082502520731c626ac8cf575 caused this.

I think specifically
```ruby
                  built_product_dir.join(File.basename(resource_path, extname)).sub_ext(output_extname).to_s
```

**Comment by dnkoutso** [comment]
fixed for 1.10.1 which will ship soon.
